src/main/groovy/com/google/appengine/task/appcfg/StageTask.groovy

```diff
@@ -32,6 +32,10 @@ class StageTask extends AppConfigTaskTemplate {
     void executeTask() {
         ant.delete(dir: getStagedAppDirectory())
         super.executeTask()
+
+        // TODO: see if we can move this into appcfg
+        // remove datastore-indexes.xml if we need to (fail quietly)
```

Comment:
Yeah, because the devappserver that does local runs actually tries to look for datastore-indexes.xml and index.yaml, since stage is creating index.yaml, we need to remove datastore-indexes.xml.